### ISSUE TITLE
Quit working in YouTube.

### ISSUE THREAD

**Issue by BG1959** [austencm/youtube-auto-like#76]
I tried restarting, refreshing, etc. and nothing worked.  

__________________________
### Log
Thu Apr 29 2021
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36 Edg/90.0.818.49
YouTube variant: material
navigated: https://www.youtube.com/watch?v=KTJ4Z_3dicw
loading options...
...options loaded (disabled: false,like_what: all,like_when: percent)
liker initialized
not a watch page
stopped
liker initialized
not a watch page
stopped

**Comment by markygnlg** [comment]
No probs. I came here to find a fix too (via the link in the extensions option page), the info I posted above came from the main page, ie:

click on the "<> code" link on the top left of the page (to the left of the "Issues" link).

Good to know in case it happens again.

**Comment by austencm** [comment]
hey @BG1959, did you manage to get it working with the new version?

**Comment by BG1959** [comment]
It quit working again.
    On Saturday, May 22, 2021, 07:16:14 PM CDT, Austen Morgan ***@***.***> wrote:  
 
 


Closed #76.

**Comment by markygnlg** [comment]
2 points I need to point out here:
- there is now a v2.7.1 which was released about 24hrs ago as of this writing:
https://github.com/austencm/youtube-auto-like/releases

- secondly, I too found it 'not working' (with the 50% setting) but if I pressed 'F5'/refresh or opened the video in another tab (same as refreshing the page) the like button would be lit up: ie. it works but the graphics of the thumb only displays if you press refresh (including using v2.7.1 on Opera)

Update: I just tested it & it does not work. ie. v2.7.1 was behaving as described above for about 2 hours but now it is not (note: this was posted by me '3 hours' after the previous post)
